Get-help.md

```diff
@@ -0,0 +1,141 @@
+
+### GitHub
+
+If you have a question about a pull request or issue, you can also reach out to developers by at-mentioning them (e.g. `@developer-username`) in a comment and assigning them to the issue. Be sure you both at-mention and assign them! Some developers only look at their GitHub notifications (which at-mentions trigger), while others only look at what they're assigned to.
+
+You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
+
+## How to Ask Good Questions
+
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+
+### Setup-related questions
+
+**Helpful advice**: You can choose any OS you’re comfortable with. However, most contributors use Linux or MacOS. Getting Oppia up and running on Windows can be a challenge.
+
+#### Before you ask a question
+
+1. You can setup/install Oppia by visiting [[this|Installing-Oppia]] page.  Make sure you follow **all** the mentioned instructions from the beginning in a step-by-step manner.
+
+2. Make sure each step succeeds (verify it with the expected behavior if mentioned in wiki).
+
+3. In case of any unexpected behavior/errors at any step, make sure you check out our wiki on [how to troubleshoot when you are facing installation errors](https://github.com/oppia/oppia/wiki/Issues-with-installation%3F#when-you-encounter-an-installation-error).
+
+If you are still not able to fix your error, start following the section below to raise your question on [Gitter](https://gitter.im/oppia/oppia-chat#).
+
+#### How to ask a setup question
+
+**Note**: If you are stuck at Step X, we will assume all previous steps through X-1 were successful for you. In case there were any previously failed steps, kindly mention those too with their error logs.
+
+Please follow the template given below (mark x inside checkboxes to tick them) for sending your question to Gitter.
+
+```md
+**Checklist**
+- [ ]  I have followed the “Before you ask a question” section
+
+**System Information**
+- OS: (Be specific Ex: Ubuntu 20.04 or Ubuntu 20.04 VM on MacOS 11.2.1)
+
+**Steps followed**
+
+// If you encountered this error while following instructions, provide a link to the instructions and specify which step failed. Otherwise, list what steps caused the error. These should be detailed enough for someone else to follow them.
```

Comment:
Perhaps have "instructions" link to our instructions on the wiki.

Reply:
There are multiple wiki pages the contributors could have been following, so instead of linking to one, I'll update the text to refer to "a wiki page" instead of "instructions"

Reply:
done